.semaphore/semaphore.yml

```diff
@@ -70,3 +70,41 @@ blocks:
             - ./performous-app-build.sh -D
             - cd /Users/semaphore
             - cache store macports-10_15 ./local
+  - name: Ubuntu 18.04 Build (Meson)
+    task:
+      prologue:
+        commands:
+          - echo "Installing Performous dependencies..."
+          - sudo apt-get update -y
+          - sudo apt-get install -y gettext clang-9 gcc-8 g++-8
+          - sudo apt-get install -y python3-dev python3-pip
+          - sem-version python 3.8
+          - sudo pip3 install meson
+          - sudo apt-get install -y ninja-build
+          - sudo apt-get install -y libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-all-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++2.6-dev portaudio19-dev libicu-dev libopencv-dev libportmidi-dev libglm-dev libaubio-dev libcpprest-dev
```

Comment:
You can combine all `apt-get install -y` commands and then run the sem-version and pip3 command.

Reply:
ok